### ISSUE TITLE
enhancement: allow to set a custom managment interface IP as provided by containerlab

### ISSUE THREAD

**Issue by hellt** [carlmontanari/boxen#27]
currently a hardcoded management interface is used both in both boxen and vrnetlab (10.0.0.15). using a hardcoded interface prevent some external management systems to discover the nodes, as the "real" management address is seen as the one assigned by docker to a container, but when the node call-back home it reports its management addr to be 10.0.0.15 and not the one that was used during the discovery.

A proposal is for containerlab to add an argument with the allocated IP address and boxen to take this address and configure it for the management interface (as well as the hostfd qemu rules)

**Comment by carlmontanari** [comment]
seems reasonable to me, do you think there is any value in this being a "general" boxen thing (as in also in "local mode") or nah?

**Comment by hellt** [comment]
in the local mode you only touch the management during the install phase, right? there are no hostfwd qemu settings whatsoever?

**Comment by carlmontanari** [comment]
local mode we do hostfwd on mgmt port -- or in theory bridge (which I have working in a python prototype but never made it to boxen since I dont really care about that use case :))

**Comment by hellt** [comment]
soo, even though management port in local mode should be visible to the host OS as an interface, we do use hostforwarding? 
or wait, is local mode runs a VM inside a container as well?

**Comment by carlmontanari** [comment]
bridge mode never got re-implemented in go, so yeah, its all hostwfd for me (for now) (no container, just hostfwd on the Qemu instance).

**Comment by achurak** [comment]
Hello,

I assume this has never been implemented? Is there a workaround for this, even if it's manual? Cisco Nexus Dashboard absolutely refuses to onboard n9kv devices because of this problem (reported ip being different from the discovered one) nor does it support any other interfaces except mgmt0 unfortunately.

**Comment by hellt** [comment]
@achurak yes, no workaround so far as I know it

**Comment by carlmontanari** [comment]
yeah... hopefully look for some overhauls to boxen starting in the next few months... been long over due and should have some more focus/time for this I hope!

**Comment by hellt** [comment]
this will be handlede via transparent management datapath feature #61 
closing as dup
